### PR TITLE
Use buildpack in docs

### PR DESCRIPTION
**Pull request by ericbottard**
Fixes https://github.com/spring-cloud/spring-cloud-dataflow-server-cloudfoundry/issues/308

### CODE REVIEW

File: spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc [hunk 1]
```diff
@@ -69,7 +69,7 @@ deployment to fail
 
 [subs=attributes]
 ```
-cf push dataflow-server -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
+cf push dataflow-server -b java_buildpack_offline -m 2G -k 2G --no-start -p spring-cloud-dataflow-server-cloudfoundry-{project-version}.jar
```
Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: `cf buildpacks` doesn't list `java_buildpack_offline` as a known buildpack. May be it is just for the CF environment I target to. Also, not sure which buildpacks get added by default for any CF environment. May be we can use the one (`java_buildpack` ?) that comes from default?

Comment (ericbottard) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: That was one of my concern, that names are not consistent.
Tested on our GCP env.

Not sure there is one "that comes from default" actually... Maybe this is not worth the trouble after all. WDYT @markpollack ?

Comment (markpollack) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: I believe `-b java_buildpack` is the default value in a PCF install.  Let's align with that default value for the docs.

Comment (ilayaperumalg) [inline spring-cloud-dataflow-server-cloudfoundry-docs/src/main/asciidoc/getting-started.adoc]: ok, merging the PR with the change to use `java_buildpack`

**Comment by ilayaperumalg** [comment]
Merged as 6ddea7eb0a26c54e9f82cdc6d9fd56d1ca254016

**Pull request closed without merging**